app/views/validation.scala.html

```diff
@@ -45,8 +45,14 @@
                 <div id="upper-menu-title-bar"></div>
                 <div id="label-visibility-control-holder">
                     <button id="label-visibility-control-button" class="button label-visibility-control-button" title="@Messages("press.key", "H")" data-toggle="tooltip" data-placement="top">
-                        <img src='@routes.Assets.at("javascripts/SVValidate/img/HideLabel.svg")' class="label-visibility-control-button-icon" alt="Hide Label">
```

Comment:
Instead of having two HTML elements that we switch between hiding and showing, let's use the `i18next` JavaScript library instead. You should be able to just start with a single HTML element with no text or image. And then `LabelVisibilityControl.js` can do what it was doing before, which was switching out the text and image for the HTML element, just with the appropriate translation from `i18next`!